src/commands/logs.rs

```diff
@@ -108,14 +111,7 @@ pub async fn command(args: Args, json: bool) -> Result<()> {
         })
         .await?;
     } else {
-        stream_deploy_logs(deployment.id.clone(), |log| {
-            if json {
-                println!("{}", serde_json::to_string(&log).unwrap());
```

Comment:
I think this removes the ability to json format this? Unless I'm mistaken

Reply:
nevermind, working on it